numpy/random/tests/test_generator_mt19937.py

```diff
@@ -1259,6 +1270,18 @@ def test_multivariate_normal(self, method):
         assert_raises(ValueError, random.multivariate_normal,
                       mu, np.eye(3))
 
+    @pytest.mark.parametrize("method", ["svd", "eigh", "cholesky"])
+    def test_multivariate_normal_basic_stats(self, method):
+        n_s = 1000
+        mean = np.array([1, 2])
+        cov = np.array([[2, 1], [1, 2]])
```

Comment:
```
random = Generator(MT19937(self.seed))
```
Otherwise it uses the global unseeded generator defined at the top of the file. We should probably get rid of that at some point.